src/core/etl/tests/Flow/ETL/Tests/Unit/Row/Entry/XMLElementEntryTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Flow\ETL\Tests\Unit\Row\Entry;
+
+use Flow\ETL\Row\Entry\XMLElementEntry;
+use PHPUnit\Framework\TestCase;
+
+final class XMLElementEntryTest extends TestCase
+{
+    public function test_serialization() : void
```

Comment:
We need more tests here for the initialization of this DOMElement from object and string, additionally isEqual and toString since they also holds some additional logic.